test/TensorFlowRuntime/codable.swift

```diff
@@ -0,0 +1,64 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+// REQUIRES: swift_test_mode_optimize
+// REQUIRES: objc_interop
+
+import TensorFlow
+import Foundation
+import TensorFlowUnittest
+import StdlibUnittest
+
+var CodableTests = TestSuite("TensorFlowCodable")
+
+private func _testRoundTrip<T>(
+  of value: T, expectedJSON json: Data? = nil
+) where T : Codable, T : Equatable {
+  var payload: Data! = nil
```

Comment:
I don't understand why this has to be an IUO.

If you just write
```swift
var payload: Data
```
... without initializing it, the `do-catch` block is guaranteed to initialize it anyway (by definitive initialization). I think it should compile.

Reply:
Ok, I assumed that the `catch` block was trapping. Turns out it's `expectUnreachable` which doesn't return never. Then it makes sense to not compile.